charts/tekton-apps/templates/wordpress/roles.yaml

```diff
@@ -16,7 +16,7 @@ kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
  name: {{ include "tekton-apps.resourceName" (set $data "suffix" "build-pipeline-role") }}
- namespace: {{ $project.argocd.namespace }}
+ namespace: {{ include "tekton-apps.set-namespace-from-component-or-project" (dict "component" $component "project" $project) }}
```

Comment:
Shouldn't we have the same changes for `build-pipeline-role` for general components? (currently we added there possibility to define own namespace, but it won't work without it as I understand)

Reply:
Already there https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/ae0c028ab80db77651e59197b031173fa295fad5/charts/tekton-apps/templates/roles.yaml#L18